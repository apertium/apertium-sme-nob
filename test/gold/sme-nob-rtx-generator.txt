[AiTlE_q9DH0E]
Jaakko og Máret er på gårdsplassen. [/option]
[/AiTlE_q9DH0E]
[OWonglBoEVbP]
Vet du hvorfor? [/option]
[/OWonglBoEVbP]
[Qgj3s2Uy1VW6]
Men katten er i huset, katten sover. [/option]
[/Qgj3s2Uy1VW6]
[XBTSPoad1gJc]
Det er en leke. [/option]
[/XBTSPoad1gJc]
[aXUEZt_YHn39]
Har Máret også hund? [/option]
[/aXUEZt_YHn39]
[bDRQ4VP0PWaa]
Ikke, Máret #ikke har en hund, på #hun har en katt. [/option]
Nei, Máret har ikke hund, hun har katt. [/option]
[/bDRQ4VP0PWaa]
[glIcu9gZWRNs]
Så du han? [/option]
[/glIcu9gZWRNs]
[lf7ZXyOSiRwO]
Men i går var det svært kaldt! Da kunne de ikke leke ute. [/option]
[/lf7ZXyOSiRwO]
[p0ED9BccKBsd]
Jaakko har en liten hund, den er også på gårdsplassen. [/option]
[/p0ED9BccKBsd]
[rTnwJ05zqz7k]
HVOR ER JAAKKO? [/option]
[/rTnwJ05zqz7k]
[rtfC5S_yOAH0]
Hunden er svært glad nå. [/option]
[/rtfC5S_yOAH0]
[tXdMXCx-CQz4]
Jaakko er en lilten gutt og han er seks år gammel. [/option]
[/tXdMXCx-CQz4]
