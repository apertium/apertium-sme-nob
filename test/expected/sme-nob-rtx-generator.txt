[20HgeipBy8NN#0] #Han lytter nøyaktig.
[/20HgeipBy8NN]
[2AsFxSd_q8It#0] Máret tenker, at #den har #se hvor Jaakko har #gå: "Fortell #jeg hvor han er!", han sier ei mor.
[/2AsFxSd_q8It]
[49QBS5nTCoAn#0] #Han #se et bord #under og stoler #under, men Jaakko er ikke derfra.
[/49QBS5nTCoAn]
[4az6DzI2UoCs#0] #Han ser ikke #noe og #han leser.
[/4az6DzI2UoCs]
[7Hx4JwM5kbFX#0] Han prøver enda søke Jaakko.
[/7Hx4JwM5kbFX]
[9n7FXDql-dLe#0] Så #han ser ei liten hånd #også og når #han kommer nærmere, så #han ser #han et hode også!
[/9n7FXDql-dLe]
[AiTlE_q9DH0E#0] Jaakko og Máret er en gårdsplass.
[/AiTlE_q9DH0E]
[EMCu_DcTpZdh#0] Jeg ser store bøker bak huset.
[/EMCu_DcTpZdh]
[F4YGT3BDFa23#0] #Han #se #hver på stedet, men ikke han kan finne Jaakko.
[/F4YGT3BDFa23]
[FYO2dU7K_S4F#0] Og #hva Jaakko #gjøre en ved #hos?
[/FYO2dU7K_S4F]
[G5vmCm5hj-W6#0] Der #den stemme #være igjen!
[/G5vmCm5hj-W6]
[JV_evOHUnLTA#0] #Han spør en hund: "Ser Jaakko?".
[/JV_evOHUnLTA]
[KFHWfkDFyVos#0] Så #han hører en stemme, #den #komme #stor en gammel ved #avhengig av.
[/KFHWfkDFyVos]
[LDlG2Djufa9R#0] Jaakko og Máret liker å leke, #de leker alltid sammen på gårdsplassen et stort hus #utenfor.
[/LDlG2Djufa9R]
[N3EmyGF0tV8b#0] Jeg ser ei stor bok.
[/N3EmyGF0tV8b]
[NkR6dWcTJVxY#0] Jeg ser store bøker bak det røde huset.
[/NkR6dWcTJVxY]
[OWonglBoEVbP#0] Vet hvorfor?
[/OWonglBoEVbP]
[PJE8USNbWjkm#0] Kan være Jaakko?
[/PJE8USNbWjkm]
[QfoPOMULHlxZ#0] den Lille småjenta er #han lillesøster, #han er fem år #gammel.
[/QfoPOMULHlxZ]
[Qgj3s2Uy1VW6#0] Men katt er et hus, katt sover.
[/Qgj3s2Uy1VW6]
[S-jPr9xTMYJt#0] Hund #like leke det to et barn.
[/S-jPr9xTMYJt]
[SK5RR6iwHpot#0] Hvorfor #han gjør så?
[/SK5RR6iwHpot]
[Tu5i0tlrtPSC#0] Máret ler og sier: "Fant deg!".
[/Tu5i0tlrtPSC]
[VeE2bOVKhLiU#0] Når Máret #ferdig si, hun ser omkring.
[/VeE2bOVKhLiU]
[XBTSPoad1gJc#0] #Den er en leke.
[/XBTSPoad1gJc]
[__63zKs1_1Rz#0] det Er den gode været i dag, er svært varmt.
[/__63zKs1_1Rz]
[aXUEZt_YHn39#0] Har Máret også hund?
[/aXUEZt_YHn39]
[bDRQ4VP0PWaa#0] Ikke, Máret #ikke har en hund, på #hun har en katt.
[/bDRQ4VP0PWaa]
[cV3vZFz7xOOS#0] Nå #han hører det godt.
[/cV3vZFz7xOOS]
[e3n2HcJxdzsd#0] Jeg ser store bøker.
[/e3n2HcJxdzsd]
[eMYV9l1YcpOW#0] Jeg ser ei bok
[/eMYV9l1YcpOW]
[ea0_LD-ZpoNN#0] uten huset
[/ea0_LD-ZpoNN]
[gHI-JpojtCYy#0] "Ikke Máret, #ikke kan fortelle #du!", hun svarer.
[/gHI-JpojtCYy]
[gQYXApgNZ4RH#0] omkring *jandora
[/gQYXApgNZ4RH]
[glIcu9gZWRNs#0] Så #han?
[/glIcu9gZWRNs]
[gyLFWDp2Tan1#0] Må være Jaakko!
[/gyLFWDp2Tan1]
[iI9PZM-1Ppmb#0] #De mor #være et hus #i en katt sammen, #han ser et vindu ut og ser Jaakko og Máret #leke.
[/iI9PZM-1Ppmb]
[jQ19cXpeiBoP#0] #De #begge #være glad og går et hus, er en tid #spise #noe og drikke litt et vann!
[/jQ19cXpeiBoP]
[k9CPoPOixGup#0] Máret vet ikke hvor Jaakko er.
[/k9CPoPOixGup]
[lf7ZXyOSiRwO#0] Men i går #være svært #kald! #De #ikke kunne leke da ute.
[/lf7ZXyOSiRwO]
[lmHmpXXI2PDn#0] Jeg ser ei stor bok.
[/lmHmpXXI2PDn]
[p0ED9BccKBsd#0] Jaakko har en den lille #liten hund, #den #også #være nå på gården.
[/p0ED9BccKBsd]
[r95wrs1edqen#0] Máret ser ei mor et glass #avhengig av, #hun mor ler.
[/r95wrs1edqen]
[rIeKWAHnfLWF#0] Máret sitter og på #hun har hender øyne #for.
[/rIeKWAHnfLWF]
[rTnwJ05zqz7k#0] HVOR JAAKKO ER?
[/rTnwJ05zqz7k]
[rYKt-xziFTYZ#0] Han søker Jaakko: hvor #den gikk?
[/rYKt-xziFTYZ]
[rtfC5S_yOAH0#0] Hund #være svært #gledelig nå.
[/rtfC5S_yOAH0]
[scTGy9D7WOvE#0] #Ikke er en fugl #eller #noe slags #annen dyr.
[/scTGy9D7WOvE]
[sdc4dVjiYXdp#0] Selv om #han kanskje vet hvor gutt er, #han #ikke ønsker å fortelle.
[/sdc4dVjiYXdp]
[tNiYrATQN3mQ#0] #Ikke #noe får et svar, når snakker hunder!
[/tNiYrATQN3mQ]
[tXdMXCx-CQz4#0] Jaakko er en den lille smågutten og han er seks år #gammel.
[/tXdMXCx-CQz4]
[toqKiaafxvkh#0] bak huset
[/toqKiaafxvkh]
[usXVdG2H3AVG#0] Jaakko løper fort #stor en gammel ved #til, han gjemmer Máret.
[/usXVdG2H3AVG]
[wXMY0IGmmk_n#0] Máret går sakte en gård #gjennom.
[/wXMY0IGmmk_n]
[yba_4ge-5sbv#0] Men hund selvfølgelig #ikke kan snakke! Máret #ikke får et svar til spørsmålet.
[/yba_4ge-5sbv]
